Source/WebKit/UIProcess/Cocoa/XPCConnectionTerminationWatchdog.h

```diff
@@ -54,6 +54,7 @@ class XPCConnectionTerminationWatchdog {
     OSObjectPtr<xpc_connection_t> m_xpcConnection;
     RunLoop::Timer m_watchdogTimer;
     Ref<ProcessAndUIAssertion> m_assertion;
+    WeakPtr<AuxiliaryProcessProxy> m_process;
```

Comment:
Pretty sure this needs to be a Ref. Nothing keeps a process alive at this point. So your new code is not likely to run.